### ISSUE TITLE
Complete the inference file

### ISSUE THREAD

**Issue by tyagi-iiitv** [tyagi-iiitv/PointPillars#1]
The network presently outputs 6 regression targets and a classification label. We need to write code for NMS to generate final bounding boxes for predictions.

**Comment by scrouzet** [comment]
As I understand it, there is currently no way to evaluate the performance of the trained model on a validation set. I'm not sure how it is supposed to be done on Kitti 3D object detection. It seems that the testing set officially distributed does not have the labels. Do you know what are the "official splits" of the Kitti 3D object detection dataset?

**Comment by tyagi-iiitv** [comment]
That is true. However, I was talking more on the visualization perspective - that we generate the bounding boxes and try to see how effective are the predictions to start off with. Obviously, once we have some confidence that this network is working fine, we can evaluate it on the Kitti's test set.
